thoth/common/openshift.py

```diff
@@ -1427,6 +1427,24 @@ def schedule_mi_workflow(
             },
         )
 
+    def schedule_mi_kebechet_workflow(
```

Comment:
can't we extend existing method, to pass the parameter we want? I feel like mi_workflow is one and we can provide specific parameter to do something specific, WDYT?

Reply:
yes, that's correct. Will do that

Reply:
Aight, pls have a look at PR and discussion here: https://github.com/thoth-station/thoth-application/pull/692#discussion_r536031962

Kebechet workflows will be then submitted just from `mi-scheduler`